src/Timer_TEST.cc

```diff
@@ -41,6 +41,7 @@ TEST(Timer_TEST, Sequence)
 
   t.Stop();
   EXPECT_FALSE(t.Running());
+  std::this_thread::sleep_for(std::chrono::milliseconds(1));
```

Comment:
Is this addition needed?

Reply:
No, I believe the flakiness was addressed via https://github.com/ignitionrobotics/ign-common/pull/343

Removed.